iothub/device/src/Authentication/HsmAuthentication/Transport/UnixDomainSocketEndPoint.cs

```diff
@@ -5,14 +5,23 @@
 using System;
 using System.Text;
 using System.Net;
+using System.Net.Sockets;
+using System.Diagnostics;
 
 namespace Microsoft.Azure.Devices.Client.HsmAuthentication.Transport
 {
     internal sealed class UnixDomainSocketEndPoint : EndPoint
     {
-        private const int NativePathLength = 91; // sockaddr_un.sun_path at http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html, -1 for terminator
+        private const AddressFamily EndPointAddressFamily = AddressFamily.Unix;
```

Comment:
All this stuff had to be added back. I added comments below about not removing the methods that appear not to be referenced.